lib/App/Rakubrew/Build.pm

```diff
@@ -68,6 +68,11 @@ sub _get_relocatable_option {
     exit 1;
 }
 
+sub available_backends {
```

Comment:
This function was moved to `App::Rakubrew::Variables` a while ago, but I failed to change all callsites. So please just use `App::Rakubrew::Variables::available_backends()` instead of readding the function here.